composer.json

```diff
@@ -52,8 +52,9 @@
     },
     "require-dev": {
         "ext-simplexml": "*",
-        "vlucas/phpdotenv": "^4.0 | ^5.0",
+        "vlucas/phpdotenv": "^5.0",
         "symfony/process": ">=4.4 <7.0",
+        "symfony/dotenv": "^6.0",
```

Comment:
Is only `"^6.0"` compatible with these changes or could we also use `">=4.4 <7.0"`?

Reply:
Yes, please change this constraint and add support for older versions of symfony/dotenv or at least make sure that they don't cause crash.

Also please merge latest changes from 5.0 branch to your branch.

Reply:
Yeah, `>= 4.4 <7.0` is totally fine, oversight on my end. Made the change and rebased on current 5.0.